apps/mobile/src/components/Gallery/PublishGallery/PublishGalleryPreview.tsx

```diff
@@ -0,0 +1,134 @@
+import { useMemo } from 'react';
+import { View } from 'react-native';
+import { graphql, useFragment } from 'react-relay';
+import { removeNullValues } from 'shared/relay/removeNullValues';
+import { parseCollectionLayout } from 'shared/utils/collectionLayout';
+
+import { GalleryEditorTokenPreview } from '~/components/GalleryEditor/GalleryEditorTokenPreview';
+import { WhiteSpace } from '~/components/GalleryEditor/SortableTokenGrid/SortableTokenGrid';
+import { Typography } from '~/components/Typography';
+import { PublishGalleryPreviewFragment$key } from '~/generated/PublishGalleryPreviewFragment.graphql';
+
+type Props = {
+  galleryRef: PublishGalleryPreviewFragment$key;
+};
+
+const CONTAINER_WIDTH = 300;
+
+export function PublishGalleryPreview({ galleryRef }: Props) {
+  const gallery = useFragment(
+    graphql`
+      fragment PublishGalleryPreviewFragment on Gallery {
+        id
+        name
+
+        collections {
+          name
+          collectorsNote
+          layout {
+            __typename
+            sections
+            sectionLayout {
+              columns
+              whitespace
+            }
+            ...collectionLayoutParseFragment
+          }
+          tokens {
+            id
+            token {
+              ...GalleryEditorTokenPreviewFragment
+            }
+          }
+        }
+      }
+    `,
+    galleryRef
+  );
+
+  // get first section in the gallery
+  const firstSection = gallery.collections?.[0];
```

Comment:
Only get the first section of gallery for rendering gallery preview. Since it only have a small container